datalad/resources/website/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Datalad Repository</title>
+        <link rel="shortcut icon" type="image/x-icon" href=".git/datalad/web/assets/images/favicon-16x16.png" />
+        <link rel="stylesheet" type="text/css" href=".git/datalad/web/assets/css/jquery.dataTables.css">
+        <style type="text/css" class="init">
```

Comment:
Yeah, don't know where I got that snippet/idea from. The only place I could find a reference to using class="init" in datatables is [here](https://github.com/DataTables/DataTables/blob/master/examples/server_side/ids.html). 
Anyway removing it seems to be working fine, so fixed it. 
Better to remove it then to have unknown cruft ;)